torch/nn/modules/activation.py

```diff
@@ -58,6 +58,8 @@ class ReLU(Threshold):
 
     .. image:: scripts/activation_images/ReLU.png
 
+    This is not a test
```

Comment:
hmm what's this?

Reply:
whoops! was demoing workflow and that slipped in. I'll delete that!